### PR TITLE
Axis title wordwrap

### PR DESCRIPTION
**Pull request by nikgalushko**
```
valueAxis: {
    title: {
        wordWrap: "word-break",
        textOverflow: "ellipsis",
        text: "..."
    }
}
```

### CODE REVIEW

File: js/viz/docs/doccharts.js [hunk 1]
```diff
@@ -1296,7 +1308,7 @@ var dxChart = {
             * @type string
             * @default undefined
             */
-            text: undefined
+            text: undefined
```
Comment (RomanTsukanov) [inline js/viz/docs/doccharts.js:1311]: An unnecessary change

**Review by RomanTsukanov — DISMISSED** [review]

**Review by dxBeardedBear — APPROVED** [review]
